iocage_lib/ioc_start.py

```diff
@@ -918,42 +852,155 @@ def start_network_vnet_addr(self, iface, ip, defaultgw, ipv6=False):
         dhcp = self.get('dhcp')
         wants_dhcp = True if dhcp or 'DHCP' in self.get(
             'ip4_addr').upper() else False
+        wants_defaultgw = (
+            iface.rsplit('vnet')[1][0] == '0'
+            and defaultgw != 'none'
+        )
+        is_netgraph = self.get('netgraph')
 
-        if 'vnet' in iface:
+        if 'vnet' in iface and not is_netgraph:
             # Inside jails they are epairNb
             iface = f'{iface.replace("vnet", "epair")}b'
 
         if ipv6:
             ifconfig = [iface, 'inet6', ip, 'up']
             # set route to none if this is not the first interface
-            if iface.rsplit('epair')[1][0] == '0':
+            if wants_defaultgw:
                 route = ['add', '-6', 'default', defaultgw]
             else:
                 route = 'none'
         else:
             ifconfig = [iface, ip, 'up']
             # set route to none if this is not the first interface
-            if iface.rsplit('epair')[1][0] == '0':
+            if wants_defaultgw:
                 route = ['add', 'default', defaultgw]
             else:
                 route = 'none'
 
         try:
             if not wants_dhcp and ip != 'accept_rtadv':
                 # Jail side
-                iocage_lib.ioc_common.checkoutput(
-                    ['setfib', self.exec_fib, 'jexec', f'ioc-{self.uuid}',
-                     'ifconfig'] + ifconfig, stderr=su.STDOUT)
+                self.__ifconfig__(*ifconfig,
+                                  fib=self.exec_fib,
+                                  jail=f'ioc-{self.uuid}')
```

Comment:
Keep the braces aligned with the definition
```suggestion
                self.__ifconfig__(
                    *ifconfig,
                    fib=self.exec_fib,
                    jail=f'ioc-{self.uuid}'
                )
```